app/models/manageiq/providers/ansible_tower/embedded_ansible_worker.rb

```diff
@@ -0,0 +1,20 @@
+class ManageIQ::Providers::AnsibleTower::EmbeddedAnsibleWorker < MiqWorker
```

Comment:
Ok, I'll change this to:

`class ManageIQ::Providers::EmbeddedAnsible::MonitoringWorker < MiqWorker`

in manageiq/providers/embedded_ansible/monitoring_worker.rb or something.

Until, we decide to move it again... The important part is it will be in the namespace of a separate provider specific to the embedded ansible.